static/js/app.js

```diff
@@ -694,8 +694,54 @@ function toggleClass(elem) {
       item.classList.remove("is-active");
   });
   elem.target.classList.add("is-active");
+
+  const findClassName = elem.target.classList;
+  const accordionClassArray = Array.from(findClassName);
+
+  const urlRelay = window.location.href;
+  const accordionHash = window.location.hash;
+
+  if (accordionHash === ""){
+    window.location.href = urlRelay.concat("#" + accordionClassArray[1]);
+  }
+
+  else {
+    window.location.href = urlRelay.replace(accordionHash, "#" + accordionClassArray[1]);
+  }
 }
 
+function hashChangeAccordion(){
+  if (location.hash === "#use-case-shopping"){
+    useCaseTitle.forEach( item => {
+      item.classList.remove("is-active");
+    });
+    document.querySelector(".use-case-shopping").classList.add("is-active");
+  }
+
+  if (location.hash === "#use-case-social-networks"){
+    useCaseTitle.forEach( item => {
+      item.classList.remove("is-active");
+    });
+    document.querySelector(".use-case-social-networks").classList.add("is-active");
+  }
+
+  if (location.hash === "#use-case-offline"){
+    useCaseTitle.forEach( item => {
+      item.classList.remove("is-active");
+    });
+    document.querySelector(".use-case-offline").classList.add("is-active");
+  }
+
+  if (location.hash === "#use-case-gaming"){
+    useCaseTitle.forEach( item => {
+      item.classList.remove("is-active");
+    });
+    document.querySelector(".use-case-gaming").classList.add("is-active");
+  }
```

Comment:
This can be migrated to a switch case instead a stack of `if` statements. 

```
 switch (location.hash) {
    case "#use-case-shopping":
      useCaseTitle.forEach( item => {
        item.classList.remove("is-active");
      });
      document.querySelector(".use-case-shopping").classList.add("is-active");
      break;
    …
  }
```

Reply:
As a counterpoint to that (to make things more confusing :sweat_smile: ), I think I'm generally partial to a stack of `if` s tatements rather than `switch`, as there's no risk of forgetting a `break`, and the consistent structure makes it otherwise just as easy to scan :)

Reply:
@maxxcrawford @Vinnl Thank you both for your two cents...


I guess now I'm at a loss about which method to go with 🥴

Reply:
I'm keeping the `if` statements in for the time being